app/src/main/java/com/SecUpwN/AIMSICD/adapters/AIMSICDDbAdapter.java

```diff
@@ -138,10 +142,12 @@
     private String[] mTables;
     private SQLiteDatabase mDb;
     private Context mContext;
+    private SharedPreferences mPreferences;
 
     public AIMSICDDbAdapter(Context context) {
         super(context, DB_NAME, null, 1);
         mContext = context;
+        mPreferences = PreferenceManager.getDefaultSharedPreferences(mContext);
```

Comment:
Is there a reason to make this field class global? It's used only in one method(?)